### ISSUE TITLE
Downvote by Artist

### ISSUE THREAD

**Issue by lincalinca** [sigma67/ytmusicapi#218]
I want to banish certain artists from ever being listened to. At best, it looks like if I downvote all of their songs, it'll eradicate at least their existing library from ever coming up in listens, which is fine for now.

Some artists, this is fine to be done individually, where they have like 20 songs, but some, like R Kelly, there's hundreds. I want to `nope` them away, but it'll just take too long to do manually.

tl;dr: Can you add the upvote and downvote functionality, and the ability to filter whole artists to them apply the trigger to?

**Comment by sigma67** [comment]
I don't think this is even a feature of YouTube Music? You can do as you suggested by getting all songs of a particular artists and using  `rate_song` to dislike them, but for an artist downvote feature you'll need to ask Google Support (this is an unofficial project).

**Comment by lincalinca** [comment]
I know yt doesn't have an artist downvote, so downvoting all songs in bulk (i.e. systematically via api) seems the most workable solution.
